src/constants/breakpoints.ts

```diff
@@ -1,7 +1,8 @@
 export const breakpoints = {
-  desktop: 1280,
-  phone: 600,
-  tablet: 720,
+  largeDesktop: 1200,
+  desktop: 992,
+  phone: 576,
+  tablet: 768,
```

Comment:
```suggestion
  tablet: 768,
  phone: 576,
```
Just being picky but can we have them in order 😛